feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/cross_chain/setup/CrossChainSetupViewModel.kt

```diff
@@ -213,23 +214,41 @@ class CrossChainSetupViewModel @Inject constructor(
         }
     }.stateIn(viewModelScope, SharingStarted.Eagerly, defaultAmountInputState)
 
+    private val hasDestinationFeeAmountFlow = MutableStateFlow(false)
+    private val destinationFeeAmountFlow = chainAssetsManager.destinationChainIdFlow.map { _destinationChainId ->
```

Comment:
So as I can see, the destinationFeeAmountFlow will change each time the destinationChainIdFlow changed. Does the amount of transfer affects fee?

Reply:
Amount of transfer doesn't affect fee